backend2/src/db/models/subscription.py

```diff
@@ -0,0 +1,11 @@
+from sqlalchemy import Column, text, UnicodeText, TIMESTAMP, Boolean, Integer
+from sqlalchemy.dialects.postgresql import UUID
+
+from src.db.session import Base
+
+
+class Subscription(Base):
+    __tablename__ = "subscription"
+
+    user_id = Column(UUID, primary_key=True)
```

Comment:
Use [\_\_table_args\_\_](https://docs.sqlalchemy.org/en/14/orm/declarative_tables.html#declarative-table-configuration) to declare a composite primary key and foreign key in the same time.

Something like that:
```Python
__table_args__ = (
    PrimaryKeyConstraint('user_id', 'mutation'),
    ForeignKeyConstraint(['user_id'], ['user.id'], onupdate='CASCADE', ondelete='CASCADE'),
)
```

Reply:
Thanks! All fixed by #209c4afd